server/routes/forgotPassword.js

```diff
@@ -4,6 +4,15 @@ const knex = require("../utils/dbConfig");
 const jwt = require("jsonwebtoken");
 const transporter = require("../utils/nodemailer");
 
+const API_KEY = process.env.API_KEY;
+const DOMAIN = process.env.DOMAIN_NAME;
+
+const formData = require("form-data");
```

Comment:
explain why you need this formData here?
I think it is not required, can we use mailgun without this package?